src/main/webapp/app/more-feedback/more-feedback.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { JhiAlertService } from 'ng-jhipster';
+import { ComplaintService } from 'app/entities/complaint/complaint.service';
+import { Complaint, ComplaintType } from 'app/entities/complaint';
+import { Result } from 'app/entities/result';
+import { HttpErrorResponse } from '@angular/common/http';
+import { Moment } from 'moment';
+import { ComplaintResponseService } from 'app/entities/complaint-response/complaint-response.service';
+import { ComplaintResponse } from 'app/entities/complaint-response';
+
+@Component({
+    selector: 'jhi-more-feedback-form',
+    templateUrl: './more-feedback.component.html',
+    providers: [JhiAlertService],
+})
+export class MoreFeedbackComponent implements OnInit {
+    @Input() resultId: number;
+    complaintText = '';
+    alreadySubmitted: boolean;
+    submittedDate: Moment;
+    accepted: boolean;
+    handled: boolean;
+    complaintResponse: ComplaintResponse;
+
+    constructor(private complaintService: ComplaintService, private jhiAlertService: JhiAlertService, private complaintResponseService: ComplaintResponseService) {}
+
+    ngOnInit(): void {
+        this.complaintService.findByResultId(this.resultId).subscribe(
+            res => {
+                if (!res.body) {
+                    return;
+                }
+                this.complaintText = res.body.complaintText;
+                this.alreadySubmitted = true;
+                this.submittedDate = res.body.submittedTime!;
+                this.accepted = res.body.accepted;
+                this.handled = this.accepted !== undefined;
+
+                if (this.handled) {
+                    this.complaintResponseService.findByComplaintId(res.body.id).subscribe(complaintResponse => (this.complaintResponse = complaintResponse.body!));
+                }
+            },
+            (err: HttpErrorResponse) => {
+                this.onError(err.message);
```

Comment:
It would be great if we could add some kind of loading indicator here.
Otherwise if the server is under load and the request takes longer, the user will not know if the page is working.